### ISSUE TITLE
Philips Hue Dimmable Plug accessory (service 00000047-0000-1000-8000-0026BB765291) initialised without Brightness (00000008-0000-1000-8000-0026BB765291) characteristics

### ISSUE THREAD

**Issue by damianjaniszewski** [ebaauw/homebridge-hue#317]
#### Issue
After upgrade to homebridge-hue 0.8.2 release and above (tested in 0.8.2, 0.8.4, 0.8.6 and 0.8.7) Philips Hue Dimmable Plug accessory (Outlet service 00000047-0000-1000-8000-0026BB765291) is initialised without Brightness (00000008-0000-1000-8000-0026BB765291) characteristics. It is visible as Power Point type in Home iOS app and it can be changed to Light but still brightness cannot be set just on/off function.
The issue does not exist in 0.7.1 release and below.
#### Log Messages
```
Mar 19 22:48:39 xxxx homebridge[23950]: [2018-3-19 22:48:39] [Hue] Philips Hue: Philips BSB001 bridge v01041302, api v1.16.0
Mar 19 22:48:40 xxxx homebridge[23950]: HAP Warning: Characteristic 00000008-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 00000047-0000-1000-8000-0026BB765291. Adding anyway.
```

#### Debug Files
[homebridge-hue-0.7.2.json.gz](https://github.com/ebaauw/homebridge-hue/files/1827180/homebridge-hue-0.7.2.json.gz)
[homebridge-hue-0.8.2.json.gz](https://github.com/ebaauw/homebridge-hue/files/1827181/homebridge-hue-0.8.2.json.gz)

**Comment by ebaauw** [comment]
The HAP Warning is a bug in homebridge-hue, but can be ignored safely.  The Brightness characteristic is added anyways.  However, it would seem Apple's Home doesn't support Brightness on an Outlet service.  Probably best to expose this plug (and any Dimmable plug) as Light instead of as Outlet (as under v0.7.x).

**Comment by damianjaniszewski** [comment]
I guess Dimmable Plug can be safely categorized and exposed as Light because it primary use is light actually. Regarding another type of plug which is not dimmable I think Outlet service is perfect since it anyway does not have Brighness functinality and simple on/off is ok. Do you think it is possible to do this that way?

**Comment by ebaauw** [comment]
Should be fixed in v0.8.8.
